Dockerfile

```diff
@@ -28,13 +28,13 @@ RUN apt-get -qq update \
   && rm /tmp/packages-microsoft-prod.deb \
   && curl -fsSL https://download.docker.com/linux/debian/gpg | apt-key add - \
   && echo 'deb [arch=amd64] https://download.docker.com/linux/debian buster stable' >> /etc/apt/sources.list \
-  && curl -fsSL https://dl-ssl.google.com/linux/linux_signing_key.pub | gpg --dearmor -o /usr/share/keyrings/dart.gpg \
-  && echo 'deb [signed-by=/usr/share/keyrings/dart.gpg arch=amd64] https://storage.googleapis.com/download.dartlang.org/linux/debian stable main' | tee /etc/apt/sources.list.d/dart_stable.list \
+  && curl -fsSL https://storage.googleapis.com/flutter_infra_release/releases/stable/linux/flutter_linux_3.7.1-stable.tar.xz -o /tmp/flutter_linux_3.7.1-stable.tar.xz \
+  && tar xf /tmp/flutter_linux_3.7.1-stable.tar.xz
+  &&
   && apt-get update -qq \
   && apt-get install -y --no-install-recommends \
     dotnet-sdk-7.0 \
     docker-ce-cli \
-    dart \
```

Comment:
I'm wondering if this should be indeed removed, no experience ith `--no-install-recommends`.

Reply:
`--no-install-recommends` is to tell `apt` to not automatically install any recommended packages automatically to avoid bulk in the Docker image.

If you remove dart, it may cause failures for the old use cases of pub.dev target as it still assumes there to be a dart binary (this should be fine if the flutter package provides it now, I just don't know)

Reply:
Should be fine then since `dart` is still available, thanks.